### ISSUE TITLE
PWA status bar colour on iOS

### ISSUE THREAD

**Issue by atbentley** [RubenVanEldik/hackerdaily-frontend#2]
Hi there,

Thanks for putting together this nice little web app!

I’ve pinned this as a PWA on my iPhone and noticed that the status bar is using default styling. This means that the background of the status bar is solid white in light mode and solid black in dark mode.

Not a big issue but a fix would make the PWA feel more native for iPhone users.

![1C83B2B2-79B6-4F2E-8C74-95BF11F5F467](https://user-images.githubusercontent.com/5869437/103286465-763ad080-49e0-11eb-97fb-c376fbc45223.png)

![61189845-868F-4D85-9124-34ABEBBD70E4](https://user-images.githubusercontent.com/5869437/103286470-7935c100-49e0-11eb-9396-ec7fe70e908b.png)

**Comment by RubenVanEldik** [comment]
Hi Andrew!

It's great to hear that you like the app!

There are only three options to set the status bar color for iOS, `default`, `black`, and `black-translucent`. I currently use the default, but I could change it to black-translucent. The only problem is that the text will always be white, so I'll have to add a bar behind the status bar to make sure that its visible while not in dark mode.

That's the reason why I haven't done it yet, but you're right, I should try to fix this to improve the native feel.

**Comment by RubenVanEldik** [comment]
I'll try to release a fix later today. :)

**Comment by atbentley** [comment]
Oh, interesting to know! Seems like a sore spot in the PWA support from Apple.

Thanks for taking a peek at it in any case.

**Comment by RubenVanEldik** [comment]
Yeah, Apple really doesn't encourage Progressive Web Apps :p

**Comment by RubenVanEldik** [comment]
I've tried to fix this with the `black-translucent` option, but I can't get it to work right. Because it's translucent you see the text below the status bar when you scroll. But I also can't put a fixed div below it. I don't know why, but when I put a fixed div at the top this div stays below the (translucent) status bar.

It's very strange, but as far as I can tell for now there is no fix for this. Therefore, I set the status bar to just 'black' for now. This looks (in my opinion) better than the white status bar in the non-dark mode.

![IMG_BB1A24A8DE65-1](https://user-images.githubusercontent.com/25854734/103545011-40469200-4ea1-11eb-9adf-0f248c588543.jpeg)
